kornia/geometry/quaternion.py

```diff
@@ -75,11 +75,15 @@ class Quaternion(Module):
 
     """
 
-    def __init__(self, data: Tensor) -> None:
+    _data: Union[Tensor, Parameter]
+
+    def __init__(self, data: Tensor, wrap_in_parameter: bool = True) -> None:
```

Comment:
wondering if the data should we expect from user to be passed explicitly as tensor or parameter, and then have `as_parameter()` or similar? feels a bit weird the `wrap_in_parameter` argument